capella/setup/disable_semantic_browser_auto_refresh.py

```diff
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+import fileinput
```

Comment:
We may want to add a module docstring that explains, why we do the string replacement and do not modify using `lxml`.

Reply:
Do you want to write it? You have looked into lxml more deeply and could explain better why it's not a good option for us.